src/Uno.UI.Runtime.Skia.Tizen/Tizen/TizenDisplayInformationExtension.cs

```diff
@@ -0,0 +1,57 @@
+﻿using ElmSharp;
+using Tizen.System;
+using Windows.Graphics.Display;
+
+namespace Uno.UI.Runtime.Skia
+{
+	public class TizenDisplayInformationExtension : IDisplayInformationExtension
+	{
+		private DisplayInformation _displayInformation;
+		private Window _window;
+		private string _profile;
+		private int? _dpi;
+
+		public TizenDisplayInformationExtension(object owner, Window window)
+		{
+			_displayInformation = (DisplayInformation)owner;
+			_window = window;
+			_profile = Elementary.GetProfile();
+		}
+
+		public DisplayOrientations CurrentOrientation => DisplayOrientations.Portrait;
+
+		public uint ScreenHeightInRawPixels => (uint)_window.ScreenSize.Height;
+
+		public uint ScreenWidthInRawPixels => (uint)_window.ScreenSize.Width;
+
+		public float LogicalDpi => _dpi ??= GetDpi();
+
+		public double RawPixelsPerViewPixel => 1; //TODO
+
+		public ResolutionScale ResolutionScale => (ResolutionScale)(int)(LogicalDpi / 1.60f);
+
+		private int GetDpi()
+		{
+			// TV has fixed DPI value (72)
+			if (_profile == "tv")
+			{
+				return 72;
+			}
+
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
What should be used ?

Reply:
Not sure here, Xamarin.Essentials also use this and documentation didn't reveal any alternative 🤔